apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClientBuilder.cs

```diff
@@ -65,6 +56,12 @@ public int? ClientCount
         }
     }
 
+    /// <summary>
+    /// Whether the <see cref="PublisherServiceApiClient"/>s may share gRPC connections or not.
+    /// Defaults to false, meaning, each client has its own gRPC connection.
+    /// </summary>
+    public bool ClientsShareConnections { get; set; }
```

Comment:
Consider ShareClientConnections instead? I'm not massively concerned about either option though.

Reply:
Yes, I'll change to that.